### PR TITLE
s3ql 2.33 (new forumla)

### PR DESCRIPTION
**Pull request by benschumacher**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adding S3QL support to Homebrew.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Looks good, except the test but you commented that you're working on that.

**Comment by fxcoudert** [comment]
```
  * fully scope test system calls e.g. system "#{bin}/fsck_s3ql"
  * Use `system "fsck_s3ql", "local://$PWD"` instead of `system "fsck_s3ql local://$PWD"` 
```

File: Formula/s3ql.rb [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+  resource "llfuse" do
+    url "https://files.pythonhosted.org/packages/19/e7/26335a22c776c763d280c2985963cf2f31484b7d05b5d74a08433d08201d/llfuse-1.3.5.tar.bz2"
+    sha256 "6e412a3d9be69162d49b8a4d6fb3c343d1c1fba847f4535d229e0ece2548ead8"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    %w[certifi urllib3 idna chardet pycrypto defusedxml requests apsw dugong llfuse].each do |r|
+      venv.pip_install resource(r)
```
Comment (fxcoudert) [inline Formula/s3ql.rb]: What about the other resources? Are they unused?

Comment (benschumacher) [reply Formula/s3ql.rb]: Yes, in this formula. They are related to building documentation. I can pull them from the script if that's desirable.

Comment (fxcoudert) [reply Formula/s3ql.rb]: Please remove all packages that are not installed, yes.

Comment (benschumacher) [reply Formula/s3ql.rb]: Fixed.

**Comment by benschumacher** [comment]
Ping?

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @benschumacher it's in!

**Pull request closed without merging**